crates/tabby-scheduler/src/dataset.rs

```diff
@@ -270,6 +270,17 @@ lazy_static! {
                     .unwrap(),
                 ),
             ),
+            (
+                "ruby",
+                TagsConfigurationSync(
+                    TagsConfiguration::new(
+                        tree_sitter_ruby::language(),
+                        tree_sitter_rust::TAGGING_QUERY,
```

Comment:
should this line be tree_sitter_ruby::TAGGING_QUERY?

Reply:
this has been fixed in main